assets/css/edd-admin-chosen.css

```diff
@@ -246,10 +246,14 @@
 }
 
 .edd-select-chosen {
-	width: 100% !important;
+	width: 100%;
 	max-width: 300px;
 }
 
+.edd-select-chosen.edd-customer-select {
+	width: 100% !important;
+}
```

Comment:
@robincornett Is this because Chosen is initially hidden inside of a tab section?